draft-welzl-ccwg-ratelimited-increase.md

```diff
@@ -123,6 +124,19 @@ Such unlimited growth in the cwnd is therefore disallowed by the first rule.
 However, in most common congestion control mechanisms, in the absence of an indication of congestion, a cwnd that has been fully utilized during an RTT is permitted to be increased during the immediately following RTT.
 Thus, such an increase is allowed by the second rule.
 
+
+### Rate-based congestion control
+
+The present document updates congestion control specifications that use a congestion window (cwnd) to limit the number of unacknowledged packets a sender is allowed to emit. This is only one way to constrain the rate of a sender.
```

Comment:
Rather than say 'This is only one way to constrain the rate of the sender`, I suggest `Use of a congestion window variable to control sending rate is not the only mechanism available and used in practice.'

To me, it is clearer to avoid the ambiguous 'This' in the current sentence.